doc/mkdocs/Makefile

```diff
@@ -2,6 +2,9 @@
 serve: prepare_files
 	venv/bin/mkdocs serve
 
+serve_dirty: prepare_files
+	venv/bin/mkdocs serve --dirtyreload
+
```

Comment:
Didn't mean to commit this … but it doesn't hurt, right?

Reply:
I am just working on the `overwork_documentation` branch and was annoyed by the slow updates. Was not aware of this - yes, please keep it!